### DISCUSSION TITLE
ResourceExhaustedError

### DISCUSSION THREAD

**Discussion by elianalousada** [neuropoly/axondeepseg#156] [Help needed] [answered]
Hello,

When launching the training procedure, using the training_guideline jupyter notebook, I get an error message that says: 
ResourceExhaustedError (see above for traceback): OOM when allocating tensor with shape[243,1024,1024,16]
Might be due to a too big of a batch size? Or patch size actually... 

Best, 
Eliana

**Comment by mathieuboudreau** [comment, marked as answer]
As we chatted about in our Skype call today, this is likely due to too large patch size and/or because you're running on CPU. I'll get back to you after our next group meeting!
